### PR TITLE
[DOCS] Alerts related by session ID and Alerts related by process ancestry no longer beta and don't require a feature flag 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/2470.
Note: Am holding off from refreshing the screenshot of the alert details flyout because I might do that in https://github.com/elastic/security-docs/issues/2480 instead.

Preview [here](https://security-docs_2527.docs-preview.app.elstc.co/guide/en/security/master/view-alert-details.html#alert-details-insights).

### CODE REVIEW

**Review by janmonschke — APPROVED** [review]

File: docs/detections/alerts-view-details.asciidoc [hunk 1]
```diff
@@ -58,18 +58,13 @@ The Insights section provides the following details:
 
 * *Cases related to the alert* - Shows the total number and names of cases to which the alert has been added. Click a case's name to open its details.
 * *Alerts related by source event* - Shows the ten most recent alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the *Investigate in timeline* button to examine related alerts in Timeline.
-* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same Session ID, which is a unique ID for tracking a given Linux session.
+* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same Session ID, which is a unique ID for tracking a given Linux session. To use this feature, the *Include session data* setting must be enabled on your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```
Comment (joepeeples) [inline docs/detections/alerts-view-details.asciidoc:61]: Revising passive voice. Also, "Session ID" prob doesn't need to be capitalized; I don't think it's a proper name, just the ID for a given session.
```suggestion
* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the *Include session data* setting on your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

File: docs/detections/alerts-view-details.asciidoc [hunk 2]
```diff
@@ -58,18 +58,13 @@ The Insights section provides the following details:
...
 +
-beta::[]
-+
-NOTE: This feature requires a https://www.elastic.co/pricing[Platinum or Enterprise subscription]. In addition, the *Include session data* setting must be enabled on your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
+NOTE: This feature requires a https://www.elastic.co/pricing[Platinum or Enterprise subscription].
```
Comment (joepeeples) [inline docs/detections/alerts-view-details.asciidoc:63]: Having the same note for multiple items feels repetitive and disrupts the flow. I'm wondering if we could consolidate the info, something like:

> The Insights section provides the following details:
> - Cases related to the alert
> - Alerts related by source event
> 
> If you have a Platinum or Enterprise subscription, these details are also included:
> - Alerts related by session ID
> - Alerts related by process ancestry

**Review by joepeeples — COMMENTED** [review]
A few optional suggestions, LGTM!

File: docs/detections/alerts-view-details.asciidoc (hunk 2 above)
Comment (nastasha-solomon) [reply docs/detections/alerts-view-details.asciidoc:63]: Yeah, agree about the notes breaking up the flow. I like this ^

File: docs/detections/alerts-view-details.asciidoc [hunk 3]
```diff
@@ -58,18 +58,11 @@ The Insights section provides the following details:
...
-* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same Session ID, which is a unique ID for tracking a given Linux session.
-+
-beta::[]
-+
-NOTE: This feature requires a https://www.elastic.co/pricing[Platinum or Enterprise subscription]. In addition, the *Include session data* setting must be enabled on your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
 
+If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], these details are also included:
+
+* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the *Include session data* setting on your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```
Comment (jmikell821) [inline docs/detections/alerts-view-details.asciidoc:64]: Prepositions are tricky, but I like to thing of config settings are "within" a window or inside, hence `in`. But it's not crucial.  
```suggestion
* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the *Include session data* setting in your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

**Review by jmikell821 — COMMENTED** [review]
Slight optional nit -- otherwise LGTM! +1 for the simpler consolidation.

File: docs/detections/alerts-view-details.asciidoc (hunk 3 above)
Comment (nastasha-solomon) [reply docs/detections/alerts-view-details.asciidoc:64]: I think both work, but "in" sounds a bit more correct when I think about where the setting is located _within_ the policy.

**Pull request merged**